play-services-core/src/main/AndroidManifest.xml

```diff
@@ -823,6 +823,12 @@
             </intent-filter>
         </service>
 
+        <service android:name="org.microg.gms.panorama.PanoramaService">
+            <intent-filter>
+                <action android:name="com.google.android.gms.panorama.service.START" />
```

Comment:
Need to remove `<action android:name="com.google.android.gms.panorama.service.START" />` from DummyService.